veps/vep-5.md

```diff
@@ -4,20 +4,20 @@
 VEP: 5
 Title: Vitess Bugfixes
 Author: Andres Taylor
-Reviewers: Derek Perkins, Rafael Chacon, Harshit Gangal, Deepthi Sigireddi
+Reviewers: Derek Perkins, Rafael Chacon, Harshit Gangal, Deepthi Sigireddi, Matt Lord
 Status: Proposed
 Created: 2021-05-19
```

Comment:
- I think your name should go next to Andres as Author(s)
- We should change Status to Accepted
- Add another line: `Amended: <today's date>`